test/Dapr.Client.Test/InvocationHandlerTests.cs

```diff
@@ -79,47 +83,10 @@ public void TryRewriteUri_FailsForRelativeUris()
         }
 
         [Theory]
-        [InlineData(null, "http://bank", "https://some.host:3499/v1.0/invoke/bank/method/")]
```

Comment:
I disagree with removing all these tests - again, they're each verifying a unique and separate variation of the inputs that is worth knowing is covered.

Reply:
I'm not removing them, I just placed them in the class data _**TryRewriteUriWithNoAppIdRewritesUriToDaprInvokeTestData**_ and all of them are being called here.

The reason why I've done this was to make this testing class a bit more cleaner.

If you don't agree with moving the test data to that class, I can revert this change.

Reply:
Which I've now seen you want to revert back to the attributes.